recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -136,6 +137,11 @@ def _patch_sources(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "sharedlib", "CMakeLists.txt"),
                               """string(REGEX REPLACE "/MT" "/MD" ${var} "${${var}}")""",
                               "")
+        if tools.Version(self.version) >= "2.1.1":
+            # avoid explicit fPIC
+            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                                  """set_target_properties(simd PROPERTIES POSITION_INDEPENDENT_CODE 1)""",
+                                  "")
```

Comment:
not required if think since the condition is `if(NOT WIN32 AND (CMAKE_POSITION_INDEPENDENT_CODE OR ENABLE_SHARED))`